spec/hardware/disk.fmf

```diff
@@ -24,6 +24,12 @@ description: |
        # String, path to the block device representing this disk.
        block-device: "/dev/vda"
 
+       # Number or String, logical sector size
+       logical-sector-size: 512
+
+       # Number or String, physical sector size
+       physical-sector-size: 4096
```

Comment:
1. Check the `size` specification above: `string`, not `String`, full stop, and the example should show the string variant as well, e.g. `512|">= 512"`
2. note that you are adding these fields among those *tmt* might add to describe the actual provisioned HW, and as such, they would never be parsed. These are output-only, never for the user to specify, therefore the parser part would not be needed. If you wanted to add new HW requirements then these need to be moved higher, under the `driver` field.